sql/mozfun/utils/get_url_path/udf.sql

```diff
@@ -0,0 +1,20 @@
+CREATE OR REPLACE FUNCTION utils.get_url_path(url STRING)
+RETURNS STRING AS (
+  "/" || COALESCE(
+    REGEXP_EXTRACT(REPLACE(REPLACE(url, "https://", ""), "http://", ""), r"\/([^&?#]*)"),
```

Comment:
Can we foresee any other protocols than HTTP and HTTPS making it through for example `ftp://`?

Reply:
not in my use case ... future people can improve upon it ;)